ncard_app/templates/detail_views/edit_event.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'events/base.html' %}
+{% load crispy_forms_tags %}
+{% load static %}
+<link rel="stylesheet" href="{% static 'style.css' %}" type="text/css" media="screen" >
+{% block title %}Edit Award{% endblock %}
+
+{% block content%}
+    <h1 class="ncard-dark-green">Edit Award</h1><br>
+    <div class="close_position"><a href="{% url 'list-events' %}" class="aui_close">&times;</a></div>
+    <div class="form-margin">
+        <form action="{% url 'edit-events' form.instance.id %}" method=POST>
```

Comment:
'edit-events' should be 'edit-event' singular for consistency and because you're only editing one event at a time

Reply:
changed it